variables.mk

```diff
@@ -136,6 +136,8 @@ output_dir=$(sim_dir)/output/$(long_name)
 #########################################################################################
 # helper variables to run binaries
 #########################################################################################
+PERMISSIVE_ON=+permissive
```

Comment:
Why not just inline this instead of keeping around the two variables?

Reply:
This is slightly more DRY in my opinion. It should always be the same in both places and this will ensure that. If we inline it someone might change one or the other without realizing they need to change both.
That my opinion and if others think we should inline I'm ok with that.

Reply:
Sounds good.